packages/paste-codemods/package.json

```diff
@@ -15,7 +15,7 @@
   "dependencies": {
     "chalk": "4.1.2",
     "execa": "3.2.0",
-    "globby": "10.0.1",
+    "globby": "npm:globby@10.0.1",
```

Comment:
question: do we need the npm prefix here?

Reply:
Yeah to avoid issues from `manypkg`